### PR TITLE
Correct handling of renamed folders

### PR DESCRIPTION
**Pull request by stigespen**
Fixes the issue detailed at https://github.com/carljm/synology-mediamon/issues/9

### CODE REVIEW

File: mediamon.py [hunk 1]
```diff
@@ -94,6 +95,10 @@ def process_IN_MODIFY(self, event):
         if self.is_allowed_path(event.pathname, event.dir):
             self.modified_files.add(event.pathname)
 
+    def process_IN_MOVE_SELF(self, event):
+    #       if self.is_allowed_path(event.pathname, event.dir):
```
Comment (carljm) [inline mediamon.py]: why is this commented?

Comment (stigespen) [inline mediamon.py]: process_IN_MOVE_SELF is just copied from process_IN_MODIFY. Not sure if the if-statement was necessary, so I commented it out. 

Looking at the is_allowed_path function now, I think it is OK not to use it here. For instance, a file misnamed photo.jpag renamed to photo.jpeg will be indexed - and directories are ignored by is_allowed_path anyways.

I do not really know Python at all, so please correct me if wrong.

Comment (carljm) [inline mediamon.py]: is the file path received by `IN_MOVE_SELF` the old path or the new path?

Comment (stigespen) [inline mediamon.py]: Not sure, will test tomorrow.

**Comment by carljm** [comment]
Thanks for the investigation and pull request!

File: mediamon.py (hunk 1 above)
Comment (stigespen) [inline mediamon.py]: Done some testing. event.pathname is the new path.
IN_MOVE_SELF is only triggered when renaming (moving) directories. Renaming files does not trigger it.
(so changing a file from file.jpag to file.jpeg does cause it to be indexed even without this patch)
So the if-statement can be ignored, as it ignores directories anyway.

Comment (carljm) [inline mediamon.py]: Ok, thanks for investigating. Can you just remove the commented line entirely, please?

**Comment by carljm** [comment]
Great pull request, thank you! Merging.

**Pull request merged**